practice-areas/README.md

```diff
@@ -25,15 +25,15 @@ Practice area meetings should be added to the main "CivicActions" calendar for v
 
 | Name                                           | Leads                                                                                        | Slack                  | Meets                                |
 | ---------------------------------------------- | -------------------------------------------------------------------------------------------- | ---------------------- | ------------------------------------ |
-| [Accessibility](accessibility/README.md)       | Laura Flannery, Mike Gifford                                                                 | #accessibility         | Twice monthly                        |
+| [Accessibility](accessibility/README.md)       | Maggie Wachs, Laura Flannery, Mike Gifford                                                   | #accessibility         | Twice monthly                        |
 | Data Science                                   |                                                                                              | #datascience           |                                      |
-| Design                                         | Emily Ryan, Jenna Waszak, Kelly Smith, Val Runde                                             | #design                | Weekly (Ops), Weekly (Lunch & Learn) |
+| Design                                         | Jenna Waszak, Kelly Smith, Val Runde                                                         | #design                | Weekly (Ops), Weekly (Lunch & Learn) |
 | DevSecOps                                      | David Sumner, Sharjeel Aziz                                                                  | #devsecops             | Biweekly                             |
 | Diversity, Equity, Inclusion and Accessibility | Rotating                                                                                     | #celebrating-diversity | Weekly                               |
 | [Drupal](engineering/drupal/README.md)         | Andy Hawks, Daniel Mundra, Eric Napier, Jonathan Bourland, Kristian Ducharme, Patrick Koroma | #engineering-drupal    | Monthly                              |
 | Frontend                                       | Dan Goodwin, Eric Huffman, Iris Ibekwe                                                       | #engineering-frontend  | Biweekly                             |
```

Comment:
FYI, while you're making changes, we can also change the "Meets" value to Monthly for FE. Thx!

Reply:
Made the change!